apps/api/src/hyp3_api/validation.py

```diff
@@ -215,6 +215,24 @@ def check_bounding_box_size(job: dict, _, max_bounds_area: float = 4.5) -> None:
         )
 
 
+def _has_rtc_static_coverage(granule_name: str) -> bool:
+    burst_id, swath = granule_name.split('_')[1:3]
+    params = {
+        'short_name': 'OPERA_L2_RTC-S1-STATIC_V1',
+        'granule_ur': f'OPERA_L2_RTC-S1-STATIC_T*-{burst_id}-{swath}_*',
+        'options[granule_ur][pattern]': 'true',
+    }
+    response = requests.get(CMR_URL, params=params)
+    response.raise_for_status()
+    return len(response.json()['feed']['entry']) > 0
+
+
+def check_rtc_static_coverage(_, granule_metadata: list[dict]) -> None:
+    bad_granules = [g['name'] for g in granule_metadata if not _has_rtc_static_coverage(g['name'])]
```

Comment:
I don't see a `name` field in the CMR record for these granules (e.g. [this one](https://cmr.earthdata.nasa.gov/search/granules.umm_json?short_name=SENTINEL-1_BURSTS&granule_ur=S1_073251_IW2_20250413T020809_VV_EF1E-BURST)) but we can get this from the job dict itself. Also, I might take back what I said this morning about assuming one granule; it seems easy enough to support multiple granules and that would allow for the slight chance that we ever want to support multiple granules for this job type.
```suggestion
def check_rtc_static_coverage(job: dict, _) -> None:
    bad_granules = [granule for granule in job['job_parameters']['granules'] if not _has_rtc_static_coverage(granule)]
```

Reply:
Oh, didn't realize we add the `name` field as part of the `_get_cmr_metadata` function.

Reply:
But I still might recommend using the granules from the job parameters, to reduce our reliance on CMR metadata given recent discussions about switching to `umm_json` at some point.